### PR TITLE
tests : MM-T1228 CTRL/CMD+K - Open DM with user not on the current team

### PR DESCRIPTION
**Pull request by poflankov**
#### Summary
CTRL/CMD+K - Open DM with user: not on the current team

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/18523

#### Release Note
```
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                wrap(team).as('mainTeam'),
+            );
+        cy.apiInitSetup().
+            then(({user, team}) => cy.
+                wrap(user).as('otherUser').
+                wrap(team).as('otherTeam'),
+            );
+    });
+
+    it('MM-T1228 CTRL/CMD+K_1 - Open DM with user not on the current team / with ANY', function() {
```
Comment (josephbaylon) [inline e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js:28]:
```suggestion
    it('MM-T1228_1 CTRL/CMD+K - Open DM with user not on the current team / with ANY', function() {
```

File: e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        });
+
+        // # Log-in with created user
+        cy.apiLogin(this.mainUser);
+
+        // * User is found and DM channel with the user opens
+        verifyUserIsFoundAndDMOpensOnClick(this.otherUser);
+    });
+
+    it('MM-T1228 CTRL/CMD+K_2 - Open DM with user not on the current team / with TEAM', function() {
```
Comment (josephbaylon) [inline e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js:42]:
```suggestion
    it('MM-T1228_2 CTRL/CMD+K - Open DM with user not on the current team / with TEAM', function() {
```

File: e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        cy.typeCmdOrCtrl().type('K', {release: true});
+
+        // # Start typing the name of other user
+        cy.findByRole('textbox', {name: 'quick switch input'}).type(this.otherUser.username);
+
+        // # Select other user from the list
+        cy.findByTestId(this.otherUser.username).should('not.exist');
+    });
+
+    it('MM-T1228 CTRL/CMD+K_3 - Open DM with user belonging to both teams / with ANY', function() {
```
Comment (josephbaylon) [inline e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js:64]:
```suggestion
    it('MM-T1228_3 CTRL/CMD+K - Open DM with user belonging to both teams / with ANY', function() {
```

File: e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        cy.apiAddUserToTeam(this.mainTeam.id, this.otherUser.id);
+
+        // # Log-in with created user
+        cy.apiLogin(this.mainUser);
+
+        // * User is found and DM channel with the user opens
+        verifyUserIsFoundAndDMOpensOnClick(this.otherUser);
+    });
+
+    it('MM-T1228 CTRL/CMD+K_4 - Open DM with user belonging to both teams / with TEAM', function() {
```
Comment (josephbaylon) [inline e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_user_from_other_team_spec.js:81]:
```suggestion
    it('MM-T1228_4 CTRL/CMD+K - Open DM with user belonging to both teams / with TEAM', function() {
```

**Review by josephbaylon — CHANGES_REQUESTED** [review]
Thanks @poflankov !! Just title changes below

**Comment by poflankov** [comment]
@josephbaylon, fixed!

**Review by furqanmlk — APPROVED** [review]
Thanks @poflankov 👍

**Pull request merged**